### DISCUSSION TITLE
get the server ip instead of the label (v6)

### DISCUSSION THREAD

**Discussion by danielzzz** [deployphp/deployer#3126] [Help needed]
hi, 

I was looking for this in docs, but I cannot find it.  

how can I get the host ip instead of the name?

when I try:
```
{{hostname}} or get('hostname')
```
I am getting the label this server has (dev/prod etc.) but no the actual server ip.

thanks, dan

**Comment by antonmedv** [comment]
On v6 it is `real_hostname`.

On v7 it is `hostname`.

**Comment by danielzzz** [comment]
thanks a lot! maybe it's worthy adding it to the docs?  
I was hoping to find this on this page  
https://deployer.org/docs/6.x/configuration#hostname

cheers, dan

**Comment by danielzzz** [comment]
this didn't work for me unless I setup a custom variable for this with
```
->set('real_host', $host)
```
